File: TaxCalculation.java
import java.util.Scanner;
public class TaxCalculation {
    public static void main(String[] args) {
        Scanner keyboard=new Scanner(System.in);
        double  grossMonthlySalary;
        System.out.println("Enter your gross monthly salary: ");
        grossMonthlySalary=keyboard.nextDouble();
        if (grossMonthlySalary< 402.00){
            double netMonthlySalary=grossMonthlySalary;
            System.out.println("Your net monthly salary after Tax is: GH¢" + netMonthlySalary);

        }
        else if (grossMonthlySalary<512){
            double balance=grossMonthlySalary-402.00;
            double taxPaidFor402=0;
            double remainingTax=balance*5/100;
            double taxObligation=taxPaidFor402 + remainingTax;
            double netMonthlySalary=grossMonthlySalary-taxObligation;
            System.out.println("Your net monthly salary after Tax is: GH¢" + netMonthlySalary);

            
        }
            else if (grossMonthlySalary<642.00){
                double balance=grossMonthlySalary-512.00;
                double taxPaidFor512=5.5;
                double remainingTax=balance*10/100;
                double taxObligation=taxPaidFor512 + remainingTax;
                double netMonthlySalary=grossMonthlySalary-taxObligation;
                System.out.println("Your net monthly salary after Tax is: GH¢" + netMonthlySalary);

              
            
        }
        else if (grossMonthlySalary<3642.00){
            double balance=grossMonthlySalary-642.00;
            double taxPaidFor642=18.5;
            double remainingTax=balance*17.5/100;
            double taxObligation=taxPaidFor642 + remainingTax;
            double netMonthlySalary=grossMonthlySalary-taxObligation;
            System.out.println("Your net monthly salary after Tax is: GH¢" + netMonthlySalary);

           
        }
        else if (grossMonthlySalary<20037.00){
            double balance=grossMonthlySalary-3642.00;
            double taxPaidFor3642=543.5;
            double remainingTax=balance*25/100;
            double taxObligation=taxPaidFor3642 + remainingTax;
            double netMonthlySalary=grossMonthlySalary-taxObligation;
            System.out.println("Your net monthly salary after Tax is: GH¢" + netMonthlySalary);

            
            
        }
        else if (grossMonthlySalary<50000){
            double balance=grossMonthlySalary-20037.00;
            double taxPaidFor20037=4642.25;
            double remainingTax=balance*30/100;
            double taxObligation=taxPaidFor20037 + remainingTax;
            double netMonthlySalary=grossMonthlySalary-taxObligation;
            System.out.println("Your net monthly salary after Tax is: GH¢" + netMonthlySalary);
          
        }
        else if(grossMonthlySalary>50000){
            double balance=grossMonthlySalary-50000.00;
            double taxPaidFor50000=13631.15;
            double remainingTax=balance*35/100;
            double taxObligation=taxPaidFor50000 + remainingTax;
            double netMonthlySalary=grossMonthlySalary-taxObligation;
            System.out.println("Your net monthly salary after Tax is: GH¢" + netMonthlySalary);


        }

        
    }


    
}

File: TaxCalculation.py
gross_monthly_salary = float(input("Enter your gross monthly salary: "))

if gross_monthly_salary < 402:
    net_monthly_salary = gross_monthly_salary
elif gross_monthly_salary < 512.00:
    balance = gross_monthly_salary - 402
    tax_paid_for_first_402 = 0
    remaining_tax = balance * 5 / 100
    total_tax_obligation = tax_paid_for_first_402 + remaining_tax
    net_monthly_salary = gross_monthly_salary - total_tax_obligation
elif gross_monthly_salary < 642.00:
    balance = gross_monthly_salary - 512.00
    tax_paid_for_first_512 = 5.5
    remaining_tax = balance * 10 / 100
    total_tax_obligation = tax_paid_for_first_512 + remaining_tax
    net_monthly_salary = gross_monthly_salary - total_tax_obligation
elif gross_monthly_salary < 3642.00:
    balance = gross_monthly_salary - 642.00
    tax_paid_for_first_642 = 18.5
    remaining_tax = balance * 17.5 / 100
    total_tax_obligation = tax_paid_for_first_642 + remaining_tax
    net_monthly_salary = gross_monthly_salary - total_tax_obligation
elif gross_monthly_salary < 20037.00:
    balance = gross_monthly_salary - 3642.00
    tax_paid_for_first_3642 = 543.5
    remaining_tax = balance * 25 / 100
    total_tax_obligation = tax_paid_for_first_3642 + remaining_tax
    net_monthly_salary = gross_monthly_salary - total_tax_obligation
elif gross_monthly_salary < 50000.00:
    balance = gross_monthly_salary - 20037.00
    tax_paid_for_first_20037 = 4642.25
    remaining_tax = balance * 30 / 100
    total_tax_obligation = tax_paid_for_first_20037 + remaining_tax
    net_monthly_salary = gross_monthly_salary - total_tax_obligation

else:
    if gross_monthly_salary > 50000.00:
        balance = gross_monthly_salary - 50000.00
        tax_paid_for_first_50000 = 13631.15
        remaining_tax = balance * 35 / 100
        total_tax_obligation = tax_paid_for_first_50000 + remaining_tax
        net_monthly_salary = gross_monthly_salary - total_tax_obligation


# Display the result
print(f"Your net monthly salary after tax is: GH¢{net_monthly_salary:.2f}")

